### DISCUSSION TITLE
[BUG] refreshBox is not a function

### DISCUSSION THREAD

**Discussion by awais545** [ColorlibHQ/AdminLTE#3252] [Q&A]
Unable to use refreshBox as it gives error refreshBox is not a function.

Simply try calling refreshBox on some div.

**Comment by REJack** [comment]
RefreshBox is AdminLTE v2, in AdminLTE v3 you need to use CardRefresh.

**Reply by awais545** [reply]
Here it it mentioned to use refreshBox https://adminlte.io/docs/3.0/javascript/card-refresh.html -- I want to use it cardRefresh with jquery. Can you please give some example how can i achieve that.

**Reply by REJack** [reply]
Oh 😯 I forgot to change refreshBox to CardWidget, the functions are basicly the same it should work if you change it to CardWidget instead of refreshBox.

**Reply by awais545** [reply]
Can you please share an example how to utilise it with jquery.

**Reply by awais545** [reply]
I am trying as this but it saying source is not defined `$("#sample-posts").CardRefresh({ source: 'http://www.google.com' });`

**Reply by awais545** [reply]
it seems like its not working with jqueryApi at all, looking forward to have a response. thankyou

**Reply by REJack** [reply]
Which version of AdminLTE v3 you use?

**Reply by awais545** [reply]
`admin-lte@^3.0.5`

**Reply by REJack** [reply]
Try to use v3.1.0-rc, there it should work. I've reworked the jquery api stuff to work with object.
